src/Ryujinx/Input/GTK3/GTK3MouseDriver.cs

```diff
@@ -6,24 +6,24 @@
 
 namespace Ryujinx.Input.GTK3
 {
-    public class GTK3MouseDriver : IGamepadDriver
+    public class Gtk3MouseDriver : IGamepadDriver
```

Comment:
```suggestion
    public class GTK3MouseDriver : IGamepadDriver
```
Same as other comment